logic/src/commonJvmAndroid/kotlin/com.wire.kalium.logic/util/BackupUtils.kt

```diff
@@ -105,4 +110,6 @@ private fun readCompressedEntry(
     return totalExtractedFilesSize to zipInputStream.nextEntry
 }
 
+private fun isInvalidEntryPathDestination(entryName: String) = entryName.contains("../")
```

Comment:
/**
 * Verification that the entry path is valid and does not contain any invalid characters leading to write in undesired directories.
 */